ui/media/js/main.js

```diff
@@ -876,6 +889,11 @@ function createTask(task) {
         }
     })
 
+    task['useSettings'].addEventListener('click', async function() {
+        TASK_REQ_NO_EXPORT.forEach((key) => delete task.reqBody[key]) // don't restore system settings when restoring tasks
+		restoreTaskToUI(task)
```

Comment:
Please use spaces instead of tabs for indentation :)

Reply:
Yeah, I noticed that and changed the tab behavior in my text editor since then. i cleaned up all my tab tab leftovers and them some.